src/test/java/plugins/ActiveDirectoryTest.java

```diff
@@ -88,11 +88,11 @@ public void wannabe_cannot_login_to_Jenkins_after_AD_security_configured() {
         String userWannabe = ENV.getUser() + "-wannabe";
         GlobalSecurityConfig security = saveSecurityConfig(userWannabe);
         jenkins.logout();
-        jenkins.login().doLoginDespiteNoPaths(userWannabe,
+        jenkins.login().doLogin(userWannabe,
```

Comment:
seems like this should be `doSuccessfulLogin` but existing behaviour so :shrug: ?

Reply:
Yes probably